packages/app/src/app/pages/Dashboard/Content/routes/Recent/index.tsx

```diff
@@ -2,28 +2,43 @@ import React, { useEffect } from 'react';
 import { useAppState, useActions } from 'app/overmind';
 import { sandboxesTypes } from 'app/overmind/namespaces/dashboard/types';
 import { Header } from 'app/pages/Dashboard/Components/Header';
-import { VariableGrid } from 'app/pages/Dashboard/Components/VariableGrid';
+import {
+  GRID_MAX_WIDTH,
+  GUTTER,
+  VariableGrid,
+} from 'app/pages/Dashboard/Components/VariableGrid';
 import { SelectionProvider } from 'app/pages/Dashboard/Components/Selection';
 import { Helmet } from 'react-helmet';
 import { DashboardGridItem, PageTypes } from 'app/pages/Dashboard/types';
+import { Element } from '@codesandbox/components';
+import { UpgradeBanner } from 'app/pages/Dashboard/Components/UpgradeBanner';
+import { SubscriptionType, TeamMemberAuthorization } from 'app/graphql/types';
 
 export const Recent = () => {
   const {
-    activeTeam,
+    activeTeamInfo,
+    activeWorkspaceAuthorization,
     dashboard: { sandboxes },
   } = useAppState();
   const {
     dashboard: { getPage },
   } = useActions();
 
+  const activeTeamId = activeTeamInfo?.id;
+
   useEffect(() => {
     getPage(sandboxesTypes.RECENT);
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [activeTeam]);
+  }, [activeTeamId]);
```

Comment:
Is there a difference between the time when `activeTeam` is set the one where `activeTeamInfo` is? because `activeTeam` should === `activeTeamInfo.id`

Reply:
@alexnm nope, but because I needed more info from the team, I need to get `activeTeamInfo` and not `activeTeam` which is just the id.